### ISSUE TITLE
Contrast of graphics and controls (WCAG 1.4.11a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1230]
**Request:** For active controls such as buttons and form fields: any visual information provided that is necessary for a user to identify that a control is present and how to operate it must have a minimum 3:1 contrast ratio with the adjacent colors. Also, any visual information necessary to indicate state, such as whether a component is selected or focused must also ensure that the information used to identify the control in that state has a minimum 3:1 contrast ratio. 

**Issues:**

1. https://slidewiki.org/ 
- The entries in the navigation menu have an insufficiently visible focus frame. (Effects all other pages too).

2. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 
- When focusing with the mouse, the menu button at the bottom left does not have sufficient contrast to the background.
- The buttons within the menu do not have enough contrast to the surrounding color areas when focusing.

![grafik](https://user-images.githubusercontent.com/48785136/74226390-befc7480-4cbc-11ea-9d45-dfae4ae6315a.png)

3. a. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
b. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 
-The color of the focus frame sometimes does not meet the contrast requirements of 3:1 to surrounding colored areas.
- The resulting color in mouse hovers sometimes does not meet the contrast requirements of 3: 1 to surrounding color areas.

-Examples:

![grafik](https://user-images.githubusercontent.com/48785136/74227715-5662c700-4cbf-11ea-8d0c-52706b8cb899.png)

4. https://slidewiki.org/search?keywords=accessibility&sort=score 
- See pages 1 and 3

![grafik](https://user-images.githubusercontent.com/48785136/74227818-8c07b000-4cbf-11ea-9d41-f9351914d524.png)

5. https://slidewiki.org/user/vivien.zschammer/recommendations 
- See pages 1 and 3
- The graphic button "öffne deck" does not achieve the required contrast of 3: 1 to surrounding colored areas.

![grafik](https://user-images.githubusercontent.com/48785136/74227886-ac376f00-4cbf-11ea-86b8-83d70d2cafb3.png)

6. https://slidewiki.org/addDeck 
- See pages 1 and 3
- In unfocused state the frames of the form elements do not achieve the required 3:1 contrast to the background.

7. https://slidewiki.org/contactus 
- See pages 1 and 3
- In unfocused state the frames of the form elements do not achieve the required 3:1 contrast to the background.

8. https://slidewiki.org/deck/127411/test/deck/127411 
- See pages 1 and 3
- The graphic buttons "Sub-präsentation" and "Sub-deck" do not achieve the required contrast of 3:1 to surrounding colored areas.

![grafik](https://user-images.githubusercontent.com/48785136/74228252-616a2700-4cc0-11ea-8329-336f06322e6d.png)

9. https://slidewiki.org/accessibility 
- See page 1

10. a. https://slidewiki.org/user/vivien.zschammer/settings/profile
b. https://slidewiki.org/user/vivien.zschammer/stats 
- See pages 1 and 3
- In unfocused state the frames of the form elements do not achieve the required 3:1 contrast to the background.

**Hints:** 
- Attention should be paid to a uniform focus emphasis.
- A sufficiently contrasted mouse focus could also be established (example: https://www.dzblesen.de/)

**Comment by aoelen** [comment]
W.r.t. the first issue. Is this related to the black focus color for the header items? Or do you mean the focus of the elements in the user menu on the right (with items Decks, Playlist etc.)?

**Comment by VivienDZB** [comment]
The black focus of the header items is alright. I mean the entries in the user menu.

**Comment by aoelen** [comment]
Ok, thank you!

**Comment by aoelen** [comment]
To make the focus indicators more uniform across SlideWiki, the following colors are used:
* #306CA1 (dark blue) as general focus color
* #000000 (black) as alternative focus color 

Also the focus is now outside each element (instead of having an inner border) 

![Screen Shot 2020-07-14 at 14 56 32](https://user-images.githubusercontent.com/1567948/87428480-b0efae80-c5e2-11ea-812d-e02fa21f4047.png)

![Screen Shot 2020-07-14 at 14 56 39](https://user-images.githubusercontent.com/1567948/87428481-b1884500-c5e2-11ea-8f4f-dbeb4f20d98f.png)

The improvements will be released soon
